<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>标准</string>
     </property>
    </item>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>820</width>
      <height>395</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="separator"/>
    <addaction name="exitaAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <widget class="QMenu" name="menu_D">
     <property name="title">
      <string>字体(D)</string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underlineAction"/>
    </widget>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="leftAlignAction"/>
     <addaction name="centerAction"/>
     <addaction name="rightAlignAction"/>
     <addaction name="justifyAction"/>
    </widget>
    <addaction name="menu_D"/>
    <addaction name="menu"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(W)</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="tileAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="printAction"/>
   <addaction name="separator"/>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
   <addaction name="separator"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underlineAction"/>
   <addaction name="separator"/>
   <addaction name="leftAlignAction"/>
   <addaction name="centerAction"/>
   <addaction name="rightAlignAction"/>
   <addaction name="justifyAction"/>
   <addaction name="separator"/>
   <addaction name="colorAction"/>
  </widget>
  <action name="newAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-ImageProcessor-Desktop_Qt_5_14_2_MinGW_64_bit-Debug/new.png</normaloff>../build-ImageProcessor-Desktop_Qt_5_14_2_MinGW_64_bit-Debug/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建一个新文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为(A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>打印(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="exitaAction">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>重做(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="text">
    <string>倾斜(I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="underlineAction">
   <property name="text">
    <string>下画线(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="leftAlignAction">
   <property name="text">
    <string>左对齐(L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="text">
    <string>居中(E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="rightAlignAction">
   <property name="text">
    <string>右对齐(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="justifyAction">
   <property name="text">
    <string>两端对齐(J)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(C)</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭(O)</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
  </action>
  <action name="tileAction">
   <property name="text">
    <string>平铺(T)</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>层叠(C)</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>下一个(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>前一个(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>关于Qt 5(Q)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
